docs/src/ExtendingCLIMA/Atmos/AtmosModel.md

```diff
@@ -155,10 +155,10 @@ model subcomponent pages to view the possible options for each subcomponent.
     param_set::AbstractParameterSet;
     orientation::O = FlatOrientation(),
     ref_state::RS = HydrostaticState(
-        LinearTemperatureProfile(
+        DecayingTemperatureProfile(
             FT(200),
```

Comment:
```suggestion
            FT(290),
```
(or 280 K, but 290 K is closer to Earth's global mean). As commented on below, it would be best to make these CLIMAParameters (Earth parameters to be specific).